jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/DummyWebElement.java

```diff
@@ -0,0 +1,112 @@
+package org.jbehavesupport.core.internal.web;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.Rectangle;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * DummyWebElements is used as WebElement representation of url and title of page
+ * It allows web steps to work with @url and @title default elements
+ */
+public class DummyWebElement implements WebElement {
```

Comment:
My 2 cents - personally I would prefer to use exceptions (NotSupportedOperationException?) instead of default dummy values

Reply:
It make more sence to me too :)